File: NewFaceRecognitionCS/Form1.cs
using System;
using System.Data;
using System.Windows.Forms;
using Microsoft.VisualBasic;
using MySql.Data.MySqlClient;

namespace NewFaceRecognition
{
    public partial class LOGIN
    {
        #region Declares
        private MySqlDataAdapter conAdpater;
        private DataTable dataTable;

        public LOGIN()
        {
            InitializeComponent();
        }

        private void btnLogin_Click(object sender, EventArgs e)
        {
            DBConnection.ConnDB();

            try
            {
                DBConnection.sql = "Select * FROM accounts WHERE username = '" + txtuser.Text + "' And password = '" + txtpass.Text + "'";
                DBConnection.cmd = new MySqlCommand(DBConnection.sql, DBConnection.conn);
                DBConnection.dr = DBConnection.cmd.ExecuteReader();
                DBConnection.dr.Read();

                if (string.IsNullOrEmpty(txtuser.Text) | string.IsNullOrEmpty(txtpass.Text))
                {
                    Interaction.MsgBox("Please fill ip the empty fields");
                    return;
                }
                // Redirect to Admin Account/Register,Update, Search Employee
                if (DBConnection.dr.HasRows == true)
                {
                    string lvl = DBConnection.dr["user_type"].ToString();
                    if (lvl == "admin")
                    {
                        Hide();
                        My.MyProject.Forms.RegisterEmployee.Show();
                    }

                    // Redirect to Time IN/OUT OF EMPLOYEES
                    else if (lvl == "user")
                    {
                        Hide();
                        My.MyProject.Forms.TimeInOut.Show();
                    }
                }
                else
                {
                    Interaction.MsgBox("LOGIN FAILED! Incorrect Username or Password", MsgBoxStyle.Critical);
                    txtuser.Clear();
                    txtpass.Clear();
                    txtuser.Focus();
                }
            }
            catch (Exception ex)
            {
                Interaction.MsgBox(ex.ToString());
            }
        }

        private void btnExit_Click(object sender, EventArgs e)
        {
            Application.Exit();
        }

        #endregion
    }
}

File: NewFaceRecognitionCS/Form1.vb
Imports MySql.Data.MySqlClient
Public Class LOGIN
#Region "Declares"
    Dim conAdpater As MySqlDataAdapter
    Dim dataTable As DataTable

    Private Sub btnLogin_Click(sender As Object, e As EventArgs) Handles btnLogin.Click
        ConnDB()

        Try
            sql = "Select * FROM accounts WHERE username = '" & txtuser.Text & "' And password = '" & txtpass.Text & "'"
            cmd = New MySqlCommand(sql, conn)
            dr = cmd.ExecuteReader
            dr.Read()

            If txtuser.Text = "" Or txtpass.Text = "" Then
                MsgBox("Please fill ip the empty fields")
                Exit Sub
            End If
            ' Redirect to Admin Account/Register,Update, Search Employee
            If dr.HasRows = True Then
                Dim lvl As String = dr("user_type").ToString
                If lvl = "admin" Then
                    Me.Hide()
                    RegisterEmployee.Show()

                    ' Redirect to Time IN/OUT OF EMPLOYEES
                ElseIf lvl = "user" Then
                    Me.Hide()
                    TimeInOut.Show()
                End If
            Else
                MsgBox("LOGIN FAILED! Incorrect Username or Password", MsgBoxStyle.Critical)
                txtuser.Clear()
                txtpass.Clear()
                txtuser.Focus()
            End If
        Catch ex As Exception
            MsgBox(ex.ToString)
        End Try
    End Sub

    Private Sub btnExit_Click(sender As Object, e As EventArgs) Handles btnExit.Click
        Application.Exit()
    End Sub

#End Region
End Class
